### PR TITLE
Create de_DE.lang

### PR DESCRIPTION
**Pull request by ReneMuetti**
Added German Translation

### CODE REVIEW

**Comment by TheCricket** [comment]
Anyone know german?

**Comment by ReneMuetti** [comment]
I have made the translation in the last 2 days.
I would therefore like to ask for all available here

**Comment by tterrag1098** [comment]
Maybe @Vexatos can verify that this is a good translation? :D

File: src/main/resources/assets/chisel/lang/de_DE.lang [hunk 1]
```diff
@@ -0,0 +1,1669 @@
...
+
+#Entity localizations
+entity.chisel.snowman.name=Schneegolem
+
+#Item localizations
+item.chisel.chisel_iron.name=Meißel
+
+item.chisel.chisel.desc.gui=Rechts-Klick um die GUI zu öffnen
+item.chisel.chisel.desc.lc1=Links klicken, um einen Block in der Welt zu meiseln
+item.chisel.chisel.desc.lc2=Can target a block by leaving it in the inventory
```
Comment (Vexatos) [inline src/main/resources/assets/chisel/lang/de_DE.lang]: You forgot to translate this.

File: src/main/resources/assets/chisel/lang/de_DE.lang [hunk 2]
```diff
@@ -0,0 +1,1669 @@
...
+#Stat localizations
+stat.blockChiseled=Gemeißelte Blöcke
+
+#Entity localizations
+entity.chisel.snowman.name=Schneegolem
+
+#Item localizations
+item.chisel.chisel_iron.name=Meißel
+
+item.chisel.chisel.desc.gui=Rechts-Klick um die GUI zu öffnen
```
Comment (Vexatos) [inline src/main/resources/assets/chisel/lang/de_DE.lang]: You might want to add a comma here, and generally use `Rechtsklick`.

**Comment by Vexatos** [comment]
The translation is decent, but there are way too many free spaces (things like `Andesit Säule` instead of `Andesitsäule`); unfortunately, I won't be able to make a pull request to fix it due to be being on vacation. I'm sorry.

**Pull request merged**